### ISSUE TITLE
Dragon Ball Fighterz memory leak is back after applying update and dlc (Android)

### ISSUE THREAD

**Issue by brujo5** [yuzu-emu/yuzu#10780]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Affected Build(s)

Early Acces bcdd35e8be

### Description of Issue

As the title says, the memory leak is back after applying update and dlc
the game crashes after a short time of fighting

### Expected Behavior

this shouldn't happen

### Reproduction Steps

just play a game, and in the middle of the fight the game crashes

### Log File

[yuzu_log_dbz.txt](https://github.com/yuzu-emu/yuzu/files/11750647/yuzu_log_dbz.txt)

### System Configuration

CPU: sd860
GPU/Driver: Adreno 640/ Turnip driver (vk 1.3.251)
RAM: 8gb
OS: android 13

**Comment by FearlessTobi** [comment]
How do you know this is a memory leak?

**Comment by brujo5** [comment]
> How do you know this is a memory leak?

The RAM begins to consume massively until it crashes.  I have 5 GB of free RAM and there comes a time when it starts to consume quickly 5-4-3-2-1 the game crashes.
this bug also presents it in egg ns.  with a statistics meter I realize the consumption for example I use scene 5 app

**Comment by brujo5** [comment]
the game became unplayable with the latest update today, even now with the base game I get the problem mentioned above

**Comment by lorram666** [comment]
yes its memory leak, i have the same problem here... i have 16gb ddr5 and i play a lot of heavy game with no bugs and using less tham 60% memory, but in DB Fighter Z, when i use some special attacks the screen freezes and memory starts to go fast to 100% till yuzu crashes. Happens with dlc/updates or even with base game.

steps to reproduce :
1 -  Go to training tutorial mode - training special attacks with goku vs kuririn.
2- procede with the tutorial teaching you how to use the special attacks till meteor attack skill.
3 - Use meteor attack skill ( →↓← + R ) and memory leak.
